### PR TITLE
Update exceptional_operations.md

### PR DESCRIPTION
**Pull request by jhdalek55**
I changed references from "pages" or "documents" to "section" to reflect the move to a hard copy. Also it is a proper name of sorts, so it needs to be capitalized.

Is "rollback" one word or two? I believe it's one word if you are describing the attack, but if you are describing the need to roll back a file, it should be two, correct?

### CODE REVIEW

**Comment by jhdalek55** [comment]
The second commit links the FAQ page to the "Customizing Uptane" page.

File: exceptional_operations.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ By default, Uptane does not allow updates to be rolled back and enforces this ac
 
 **Figure 1.** *Uptane prevents rollback attacks by rejecting older: (1) metadata files, and / or (2) images.*
 
-There are at least two ways to allow rollbacks, each with different advantages and disadvantages.
+There are at least two ways to allow updates to roll backs, each with different advantages and disadvantages.
```
Comment (patrickvacek) [inline exceptional_operations.md:20]: "rollbacks" seems better to me than "roll backs". I think the original phrase here was fine, but I could see room for improving the clarity.

File: exceptional_operations.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ By default, Uptane does not allow updates to be rolled back and enforces this ac
...
+In the first option, an OEM MAY choose to never increment the release counters of images (see Figure 2). Uptane will accept any new image associated with a release counter, as long as it is equal to the release counter of the previous image in the previous Targets metadata file. If release counters are never incremented, then all images would have the same release counters. In this situation, an ECU would accept the installation of any compatible image referred to in the new Targets metadata. (See the [Security considerations](https://uptane.github.io/deployment-considerations/security_considerations.html) section.)
```
Comment (patrickvacek) [inline exceptional_operations.md:22]: +1. I think there are several places where we refer to "documents" where we probably mean "page" or "section".

Comment (jhdalek55) [reply exceptional_operations.md:22]: I've been trying to pull those as I see them. I'll keep

File: exceptional_operations.md (hunk 1 above)
Comment (jhdalek55) [reply exceptional_operations.md:20]: On second thought I'm going to keep this as is.  I think it reads better
```suggestion
There are at least two ways to allow rollbacks, each with different advantages and disadvantages.
```

**Comment by patrickvacek** [comment]
> Is "rollback" one word or two? I believe it's one word if you are describing the attack, but if you are describing the need to roll back a file, it should be two, correct?

Yeah, I think it's one word as a noun and two words as a verb.

**Review by patrickvacek — APPROVED** [review]

**Pull request merged**